impeller/entity/entity_pass.cc

```diff
@@ -304,6 +304,14 @@ bool EntityPass::Render(ContentContext& renderer,
       renderer.GetContext()->capture.GetDocument(kCaptureDocumentName);
 
   renderer.GetRenderTargetCache()->Start();
+  fml::ScopedCleanupClosure reset_state([&renderer]() {
+    renderer.GetLazyGlyphAtlas()->ResetTextFrames();
+    renderer.GetRenderTargetCache()->End();
+    renderer.GetTransientsBuffer().Reset();
+#if IMPELLER_ENABLE_3D
+    renderer.GetSceneContext()->GetTransientsBuffer().Reset();
```

Comment:
I think the best way to do this for Scene would be a scoped cleanup in [`Scene::Render`](https://github.com/flutter/engine/blob/0b5b6e802fdca92bfd6b72c502a29bf53cfc844d/impeller/scene/scene.cc#L33) instead?

Reply:
Done